sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/bitwiseAggregates.scala

```diff
@@ -97,3 +97,42 @@ case class BitOrAgg(child: Expression) extends DeclarativeAggregate with Expects
 
   override lazy val evaluateExpression: AttributeReference = bitOr
 }
+
+@ExpressionDescription(
```

Comment:
This is a dumb question, but, where is the actual bitwise-xor implemented? I just can't see it. For my own info.

Reply:
We can implement `updateExpressions`,  `mergeExpressions ` logically, and they would be eventually codegened in `*Exec`s like `HashAggregateExec`

Reply:
@srowen we can treat `Expression` as a DSL, then this function is implemented by the DSL.